readthedocs_build/config/test_config.py

```diff
@@ -312,9 +328,9 @@ def it_gets_set_correctly():
         assert build['formats'] == ['pdf']
 
     def formats_can_be_empty():
-        build = get_build_config({'formats': ['none']})
+        build = get_build_config({'formats': []})
         build.validate_formats()
-        assert build['formats'] == ['none']
+        assert 'formats' not in build
```

Comment:
Don't we need test cases here for?

- check all possible values
- check multiple elements in the `formats` field

Reply:
Probably also another one with `null`.